app/webpacker/components/Persons/NewPersonForm/NewPersonForm.jsx

```diff
@@ -24,7 +25,7 @@ function NewPersonForm({
   const [semiId, setSemiId] = useInputState('');
   const [wcaId, setWcaId] = useInputState('');
   const [name, setName] = useInputState('');
-  const [countryIso2, setCountryIso2] = useInputState('');
+  const [countryIso2, setCountryIso2] = useState('');
```

Comment:
See other comment below, changing this to a "plain" state isn't necessary anymore, you can keep it as `useInputState` as it was.